_includes/payment-order-after-payment.md

```diff
@@ -0,0 +1,115 @@
+## Introduction
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)